src/redux/modeRedux.js

```diff
@@ -0,0 +1,32 @@
+/* selectors*/
+export const getMode = ({ mode }) => mode;
+
+// action name creator
+const reducerName = 'modeReducer';
+const createActionName = name => `app/${reducerName}/${name}`;
+
+// actions types
+export const CHANGE_MODE = createActionName('CHANGE_MODE');
+
+// action creators
+export const createActionChangeMode = payload => ({
+  payload: { tablet: 'tablet', mobile: 'mobile', desktop: 'desktop' },
```

Comment:
Nie rozumiem po co ten payload jest tutaj przekazywany, po co w `reducer` korzystamy z payload zamiast po prostu zrobić `return 'mobile'`

Reply:
Wgl to `tablet` `mobile` `desktop` powinny być w stałch, żeby nie pisać w kilku miejscach aplikacji stringów tylko używać stałych